lib/valid_email2/address.rb

```diff
@@ -6,10 +6,15 @@ module ValidEmail2
   class Address
     attr_accessor :address
 
-    PROHIBITED_DOMAIN_CHARACTERS_REGEX = /[+!_\/\s'`]/
     DEFAULT_RECIPIENT_DELIMITER = '+'.freeze
     DOT_DELIMITER = '.'.freeze
 
+    self.class.attr_accessor :prohibited_domain_characters_regex
+
+    def self.reset
+      self.prohibited_domain_characters_regex = /[+!_\/\s'`]/
+    end
+
```

Comment:
What do you think about replacing this with:

```ruby
def self.prohibited_domain_characters_regex
  @prohibited_domain_characters_regex ||= PROHIBITED_DOMAIN_CHARACTERS_REGEX
end

def self.prohibited_domain_characters_regex=(val)
  @prohibited_domain_characters_regex = val
end
```

Then we don't need the reset method, and it's a bit more idiomatic ruby.

Reply:
I my head I wanted to use the `attr_accessor` method. But your way is clearer, really. I updated the PR.